pyproject.toml

```diff
@@ -22,3 +22,4 @@ testpaths = [
 markers = [
     'integration: tests which run servers and try to connect to them'
 ]
+asyncio_mode = 'strict'  # 'auto' seems to conflict with pytest-tornasync
```

Comment:
Makes sense, this async means tests need an explicit `pytest.mark.asyncio`? Need to check that relevant tests are decorated correctly otherwise they will be skipped from here on.